db/migrate/20221122131303_add_users_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null:false, foreign_key:true
+  end
```

Comment:
- [ ]  as you can see in the diagram it should be `author_id`not `user_id ` so please make sure to fix that  by trying to change the name of the column you can do it like in the below example: 
```ruby
 add_reference :(table name) :author, foreign_key: { to_table: 'users' }
 ```